common4j/src/main/com/microsoft/identity/common/java/cache/MsalCppOAuth2TokenCache.java

```diff
@@ -95,7 +95,22 @@ private MsalCppOAuth2TokenCache(final IPlatformComponents commonComponents,
     // Suppressing unchecked warning as the return type requiring generic parameter which is not provided
     @SuppressWarnings(WarningType.unchecked_warning)
     public static MsalCppOAuth2TokenCache create(@NonNull final IPlatformComponents platformComponents) {
-        final MsalOAuth2TokenCache msalOAuth2TokenCache = MsalOAuth2TokenCache.create(platformComponents);
+        return create(platformComponents, false);
+    }
+
+    /**
+     * Factory method for creating an instance of MsalCppOAuth2TokenCache
+     * <p>
+     * NOTE: Currently this is configured for AAD v2 as the only IDP
```

Comment:
What exactly is specific to AAD v2 here?

Reply:
I agree this is unneeded, copy-pasted from the original one but I've deleted it in both.